### PR TITLE
Add SwiftFormat

### PR DESCRIPTION
**Pull request by jshier**
### Goals :soccer:
This PR adds SwiftFormat with a style similar to existing style. it also updates our GitHub action for Xcode 11.3.

### CODE REVIEW

File: Source/Request+AlamofireImage.swift [hunk 1]
```diff
@@ -221,9 +220,9 @@ extension DataRequest {
     @discardableResult
     public func responseImage(
         queue: DispatchQueue = .main,
-        completionHandler: @escaping (AFDataResponse<Image>) -> Void)
-        -> Self
-    {
+        completionHandler: @escaping (AFDataResponse<Image>) -> Void
+    )
+        -> Self {
```
Comment (cnoon) [inline Source/Request+AlamofireImage.swift]: I'm curious what all the options are for this. Is it possible to use the format it had before?

File: Tests/RequestTests.swift [hunk 2]
```diff
@@ -83,13 +83,13 @@ class DataRequestTestCase: BaseTestCase {
...
-                XCTAssertEqual(image.scale, screenScale, "image scale does not match expected value")
+            let screenScale = UIScreen.main.scale
+            let expectedSize = CGSize(width: CGFloat(100) / screenScale, height: CGFloat(100) / screenScale)
+            XCTAssertEqual(image.size, expectedSize, "image size does not match expected value")
+            XCTAssertEqual(image.scale, screenScale, "image scale does not match expected value")
             #elseif os(macOS)
-                let expectedSize = CGSize(width: 100.0, height: 100.0)
-                XCTAssertEqual(image.size, expectedSize, "image size does not match expected value")
+            let expectedSize = CGSize(width: 100.0, height: 100.0)
+            XCTAssertEqual(image.size, expectedSize, "image size does not match expected value")
```
Comment (cnoon) [inline Tests/RequestTests.swift]: Yay! Finally we don't have to keep rethinking this.

File: Source/Request+AlamofireImage.swift (hunk 1 above)
Comment (jshier) [reply Source/Request+AlamofireImage.swift]: Which part? SwiftFormat has various [options for wrapping](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wraparguments), it just depends which behavior you want to control. I think this is `preserve` wrapped arguments with `balanced` closing parens.

Comment (cnoon) [reply Source/Request+AlamofireImage.swift]: I played around with it and the `--closingparen` doesn't give us enough flexibility to format the way we have been. Probably best to just align with AF for now.

**Pull request merged**